src/v/cluster/tests/partition_balancer_planner_fixture.h

```diff
@@ -55,6 +56,7 @@ struct controller_workers {
     controller_workers()
       : dispatcher(allocator, table, leaders, state) {
         table.start().get();
+        leaders.start(std::ref(table)).get();
```

Comment:
I wonder why this is needed, at the first glance we don't use info about leaders in the balancer planner.

Reply:
yes, it isn't required, i dropped that commit